### PR TITLE
Add csv located in ./gsim/mgmpe in py package

### PR DESCRIPTION
**Pull request by daniviga**
Fixes https://ci.openquake.org/job/macos/job/master_macos_smtk/label=macos,python=python3.6/854/console

It's the only csv around (excluding tests):

```bash
$ find openquake -name \*.csv | grep -v test
openquake/hazardlib/gsim/mgmpe/akkar_coeff_table.csv
```

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -2,6 +2,7 @@ include README.md LICENSE CONTRIBUTORS.txt
 include openquake/engine/openquake.cfg
 include openquake/server/local_settings.py.pam
 include openquake/server/local_settings.py.standalone
+include openquake/hazardlib/gsim/mgmpe/*.csv
```
Comment (micheles) [inline MANIFEST.in]: I wonder if we should change that to `recursive-include openquake/hazardlib/gsim/*.csv` or even to  `recursive-include openquake/hazardlib/*.csv` ...

Comment (daniviga) [reply MANIFEST.in]: Both will include also stuff from the tests... we can use them but then we need an explicit exclude for tests stuff.

What would you like to do?

Comment (micheles) [reply MANIFEST.in]: No, the first one will not include .csv from the tests, so I vote for that.

Comment (daniviga) [reply MANIFEST.in]: I see. Will do.

**Pull request merged**